pkg/injection/codemodule/installer/symlink/symlink.go

```diff
@@ -49,6 +50,38 @@ func CreateSymlinkForCurrentVersionIfNotExists(fs afero.Fs, targetDir string) er
 	return nil
 }
 
+func CreateSymlinkForLatestVersion(fs afero.Fs, dk dynakube.DynaKube, latestVersionDir, symlinkDir string) error {
+	// MemMapFs (used for testing) doesn't comply with the Linker interface
+	linker, ok := fs.(afero.Linker)
+	if !ok {
+		log.Info("symlinking not possible", "targetDir", latestVersionDir, "fs", fs)
+
+		return nil
+	}
+
+	log.Info("creating symlink", "points-to(relative)", latestVersionDir, "location", symlinkDir)
+
+	if err := linker.SymlinkIfPossible(latestVersionDir, symlinkDir); err != nil {
+		log.Info("symlinking failed", "codemodules-version", latestVersionDir)
+
+		return errors.WithStack(err)
+	}
+
+	return nil
+}
+
+func RemoveSymlink(fs afero.Fs, symlinkPath string) error {
```

Comment:
since it's already `symlink` package, i think we can simplify for `symlink.Remove`

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/3940/commits/ee269cbac29dd6e645b309cc11b88a7e35d0dcfe

I did it for all symlink exported methods